Dockerfile

```diff
@@ -10,4 +10,5 @@ RUN apk add --update \
 
 RUN npm i -g
 
-ENTRYPOINT "npm-groovy-lint"
+ENTRYPOINT ["npm-groovy-lint"]
+CMD ["--help"]
```

Comment:
I'm not sure if we should change the default behaviour of running `npm-groovy-lint`.